### PR TITLE
[Encode] Support lowpower for HEVC

### PR DESCRIPTION
**Pull request by Tianhaol**

### CODE REVIEW

**Comment by Tianhaol** [comment]
@XinfengZhang  Could you help to review this PR?

**Comment by Tianhaol** [comment]
@XinfengZhang Could you help to review this PR?

File: encode/hevcencode.c [hunk 1]
```diff
@@ -1808,7 +1812,7 @@ static int process_cmdline(int argc, char *argv[])
             print_help();
             exit(0);
         case 1:
-            frame_bitrate = atoi(optarg);
+            frame_bitrate = atoi(optarg)*1000;
```
Comment (XinfengZhang) [inline encode/hevcencode.c:1815]: so, it was changed to K, could you help to add the description in command help

Comment (Tianhaol) [reply encode/hevcencode.c:1815]: Line 1571, I add info to indicate Kbits.
printf("   --bitrate <bitrate> Kbits per second\n");
Where else did I forget to add the description?

**Comment by Tianhaol** [comment]
@XinfengZhang Could you help to review this PR?

**Comment by XinfengZhang** [comment]
@Tianhaol could you help to change the commit message with:
title
description ?
current message is too simple

**Comment by Tianhaol** [comment]
> @Tianhaol could you help to change the commit message with: title description ? current message is too simple

I add more info to describe this PR.

**Review by XinfengZhang — APPROVED** [review]

**Pull request merged**